packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -258,12 +258,6 @@ const conf = (module.exports = convict({
       format: Boolean,
       env: 'REACT_CONVERSION_POST_VERIFY_CAD_VIA_QR_ROUTES',
     },
-    signInVerificationViaPushRoutes: {
```

Comment:
Should this flag also be removed from:
- packages/fxa-content-server/app/tests/spec/lib/router.js#L41
- packages/fxa-content-server/server/lib/routes/react-app/types.ts#L57
- packages/fxa-content-server/tests/server/routes/react-app.js#L43